### PR TITLE
added cloud_sql_proxy & mysql-client

### PR DESCRIPTION
**Pull request by monotek**
added cloud_sql_proxy & mysql-client to be able to change sql user permissions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # gcloud-kubectl-helm
 Docker image for the quaternity of [gcloud](https://cloud.google.com/sdk/docs/), [helm](https://www.helm.sh), [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/) and [SOPS](https://github.com/mozilla/sops).
 
+The image also contains [cloud_sql_proxy](https://github.com/GoogleCloudPlatform/cloudsql-proxy), [gnupg](https://pkgs.alpinelinux.org/package/v3.3/main/x86/gnupg), [mysql-client](https://pkgs.alpinelinux.org/package/edge/main/x86/mysql-client) and [yq](https://github.com/mikefarah/yq).
```
Comment (axdotl) [inline README.md]: Can we guarantee that mysql is installed in 3.3?
Because you specify no version when calling `apk add`

File: install.sh [hunk 2]
```diff
@@ -6,11 +6,10 @@ set -x
 adduser -S gkh gkh
 
 apk update
-apk add ca-certificates gnupg openssl
+apk add ca-certificates gnupg mysql-client openssl
 rm -rf /var/cache/apk/*
 
-gcloud components install kubectl -q --no-user-output-enabled
-gcloud -q components install beta
+gcloud components install -q beta cloud_sql_proxy kubectl
```
Comment (axdotl) [inline install.sh]: Why did you remove the `-no-user-output-enabled` flag?

Comment (monotek) [reply install.sh]: to see what happens in the build log.

File: README.md (hunk 1 above)
Comment (monotek) [reply README.md]: changed link to gnupg, without version...
